src/styles/SearchForm.css

```diff
@@ -0,0 +1,20 @@
+.search-form {
+  margin: auto;
```

Comment:
This doesnt make sense, ass you cant set the vertical properties of margin to auto. You could exchange it for this:
```
margin: 0 auto;
```